components/modal/ModalLogin.vue

```diff
@@ -116,6 +116,28 @@ export default {
 	},
 	methods: {
 		...mapMutations('modal', [`hideModal`]),
+		errorHandling(type) {
```

Comment:
remove type parameter and use this.modalTypeProps instead - no need for new variable